doc/lightning-fundchannel_cancel.7.md

```diff
@@ -28,7 +28,12 @@ RETURN VALUE
 On success, returns confirmation that the channel establishment has been
 canceled.
 
-On failure, returns an error.
+On error the returned object will contain `code` and `message` properties,
+with code being one of the following:
```

Comment:
Inconsistent: You indicate "`code`" earlier but only "code" on this line.  Also applies to most of the rest as well.

Reply:
Sharp eyes, amended, thanks!